lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -132,8 +132,10 @@ def respond_to_missing?(meth, *args)
 
       # Uses ActiveRecord's store method to methodize new attribute keys in extra_attributes
       def store_attributes
+        # Do not overwrite accessors for existing columns
         @extra_attributes_hash = extra_attributes
-        self.class.store_accessor(:extra_attributes, @extra_attributes_hash.keys)
+        column_attributes = PolicyElement.column_names.map(&:to_sym)
+        self.class.store_accessor(:extra_attributes, @extra_attributes_hash.except(column_attributes).keys)
```

Comment:
should there be a validation or something that extra_attributes doesn't have column attributes in it in the first place?